internal/injector/builtin/yaml/go-main.yml

```diff
@@ -19,15 +19,27 @@ aspects:
               - signature: {}
     advice:
       - inject-declarations:
+          links:
+            - gopkg.in/DataDog/dd-trace-go.v1/internal/civisibility/integrations/gotesting
           imports:
             tracer: gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer
+            testdeps: testing/internal/testdeps
           # Note: it is valid to have multiple func init() in a single compile unit (e.g, `.go` file), in which case
           # they get executed in declaration order. This means it's okay for us to add a new init function if there is
           # already one in the file, but as it currently is appended AFTER all other declarations in the file, it means
           # that it will be executed last (tracing contents of previous init functions will not be possible).
           template: |-
+            //go:linkname __dd_civisibility_isCiVisibilityEnabled gopkg.in/DataDog/dd-trace-go.v1/internal/civisibility/integrations/gotesting.isCiVisibilityEnabled
```

Comment:
I'm curious why all these functions are internal & un-exported from the tracer library. They feel like they'd be useful or even necessary for someone to implement CI viz for a custom testing harness?

Reply:
We will do that but in the future, we want to add a manual api to the library so customer could write their own "instrumentation" on their testing framework of preference, but we are not yet there, one of the pushback is thinking everything ahead to avoid breaking changes on dd-trace-go.

So the plan is to have a few instrumentations to test this internal api first.